src/vcpkg/vcpkglib.cpp

```diff
@@ -41,19 +41,18 @@ namespace vcpkg
 
         const auto updates_dir = paths.vcpkg_dir_updates;
 
-        std::error_code ec;
-        fs.create_directory(paths.installed, ec);
-        fs.create_directory(paths.vcpkg_dir, ec);
-        fs.create_directory(paths.vcpkg_dir_info, ec);
-        fs.create_directory(updates_dir, ec);
+        fs.create_directory(paths.installed, VCPKG_LINE_INFO);
```

Comment:
Does `create_directory()` not fail if the dir already exists?

Reply:
Correct. http://eel.is/c++draft/fs.op.create.directory#1

>If mkdir fails because p resolves to an existing directory, no error is reported.